Source/Laser/LaserParticleContainer.cpp

```diff
@@ -610,7 +610,7 @@ void
 LaserParticleContainer::ComputeWeightMobility (Real Sx, Real Sy)
 {
     constexpr Real eps = 0.01;
-    constexpr Real fac = 1.0/(2.0*MathConst::pi*PhysConst::mu0*PhysConst::c*PhysConst::c*eps);
+    constexpr Real fac = 1.0_rt / (2.0_rt * MathConst::pi * PhysConst::mu0 * PhysConst::c * PhysConst::c * eps);
```

Comment:
since this is a `constexpr` one could potentially leave the calculation at compile time in double and only cast down the result to `Real`.